web-marketplace/src/components/organisms/UserSellOrders/UserSellOrders.Toolbar.tsx

```diff
@@ -51,20 +70,16 @@ export const UserSellOrdersToolbar = ({
         </Subtitle>
         <div className="flex sm:flex-row flex-col sm:items-center items-end">
           <div className="sm:order-2 order-1">
-            {hasTradableCredits ? (
+            {canStartSellFlow ? (
               <CreateButton
-                hasTradableCredits={hasTradableCredits}
+                hasTradableCredits={true}
                 setIsSellFlowStarted={setIsSellFlowStarted}
               />
             ) : (
-              <InfoTooltip
-                arrow
-                placement="top"
-                title={_(msg`You have no tradable credits that can be sold.`)}
-              >
+              <InfoTooltip arrow placement="top" title={tooltipText}>
                 <div>
                   <CreateButton
-                    hasTradableCredits={hasTradableCredits}
+                    hasTradableCredits={false}
                     setIsSellFlowStarted={setIsSellFlowStarted}
                   />
                 </div>
```

Comment:
the create button should just be hidden in case the user is not allowed to create sell orders for the org
see dev note next to https://www.figma.com/design/L3VpDxt1PXTATPbSY0Z7DV/Organizations--org-member-permissions--and-project-collaborator-permissions?node-id=5697-460817&m=dev